install_config/web_console_customization.adoc

```diff
@@ -334,6 +334,38 @@ endif::[]
 
 endif::[]
 
+[[enable-tech-preview-feature]]
+=== Enabling Features in Tech Preview
+
+Sometimes features are available in tech preview. By default, these features are
+hidden from end users.
+
+Features currently in tech preview include:
+
+* Pipelines - Enabling this will add the *Pipelines* navigation item underneath
+ the *Builds* menu.
```

Comment:
We need to show probably here that `pipelines` is what they need to put in for <feature_name> below for this feature. Maybe a table? Feature name, description, script value?